### PR TITLE
Add opentelemetry-distro package

### PR DESCRIPTION
**Pull request by goatmale**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goatmale** [comment]
@conda-forge/help-python - not sure what I did wrong - what can I do to fix the pipeline?

**Comment by goatmale** [comment]
@conda-forge/help-python please help!

File: recipes/opentelemetry-distro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```
Comment (dopplershift) [inline recipes/opentelemetry-distro/meta.yaml:20]: This should help:
```suggestion
    - pip
    - hatchling
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (dopplershift) [inline LICENSE:1]: This file should be in your recipe directory, not the top level.

File: recipes/opentelemetry-distro/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - hatchling
+  run:
+    - python >=3.7
+    - opentelemetry-api >=1.12,<2.dev0
+    - opentelemetry-instrumentation ==0.36b0
+    - opentelemetry-sdk >=1.13,<2.dev0
+
+test:
+  imports:
+    - opentelemetry_distro
```
Comment (dopplershift) [inline recipes/opentelemetry-distro/meta.yaml:30]: It's complaining that `import opentelemetry_distro`. What's the right way to test (using Python) that this package is properly installed?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**